projects/netgrif-components-core/src/lib/side-menu/content-components/new-case/abstract-new-case.component.ts

```diff
@@ -267,4 +267,10 @@ export abstract class AbstractNewCaseComponent implements OnDestroy {
             this.netVersion = '';
         }
     }
+
+    toNextStep(viewValue: any): void {
+        if (viewValue !== null && viewValue.value !== undefined) {
```

Comment:
this does not solve the case when a value is written into the autocomplete input that matches none of the options.

a unit test should be added to check if the method performs as intended

Reply:
resolved in a personal call